packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -80,22 +80,34 @@ export interface ISummarizingWarning extends ContainerWarning {
 	readonly logged: boolean;
 }
 
+/** @deprecated - Was an internally-used subset of ISummarizerRuntime, no other replacement provided */
 export interface IConnectableRuntime {
 	readonly disposed: boolean;
 	readonly connected: boolean;
 	readonly clientId: string | undefined;
+	readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
 	once(event: "connected" | "disconnected" | "dispose", listener: () => void): this;
 }
 
-export interface ISummarizerRuntime extends IConnectableRuntime {
+export interface ISummarizerRuntime {
+	readonly disposed: boolean;
+	readonly connected: boolean;
+	readonly clientId: string | undefined;
 	readonly logger: ITelemetryLogger;
 	/** clientId of parent (non-summarizing) container that owns summarizer container */
 	readonly summarizerClientId: string | undefined;
 	readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
 	disposeFn?(): void;
 	closeFn(): void;
+	once(event: "connected" | "disconnected" | "dispose", listener: () => void): this;
 }
 
+/** The subset of ISummarizerRuntime required for Summarizer run coordination */
+export type IConnectableRuntime2 = Pick<
```

Comment:
Maybe `IRunCoordinationRuntime` to be specific. IConnectableRuntime doesn't seem to covey anything meaningful.

Reply:
Removing this new thing so irrelevant now.